SConstruct

```diff
@@ -473,10 +484,14 @@ if selected_platform in platform_list:
 
     if env["target"] == "release":
         if env["tools"]:
-            print("Tools can only be built with targets 'debug' and 'release_debug'.")
-            sys.exit(255)
-        suffix += ".opt"
-        env.Append(CPPDEFINES=["NDEBUG"])
+            print(
+                "Warning: The editor can only be built with targets `debug` and `release_debug` as it relies on debug checks to work.\n"
+                "         Will use the target `release_debug` for this build."
+            )
+            env["target"] = "release_debug"
```

Comment:
Hmm, I think the lines below won't be called in this case:

```py
     elif env["target"] == "release_debug":
        if env["tools"]:
            suffix += ".opt.tools"
```

Reply:
Good catch, I replaced that `elif` with an `if`. It should work as expected now.